aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/endpoints/ITestEndpoints.java

```diff
@@ -0,0 +1,18 @@
+package aquality.tracking.integrations.core.endpoints;
+
+import aquality.tracking.integrations.core.models.Suite;
+import aquality.tracking.integrations.core.models.Test;
+
+import java.util.List;
+
+public interface ITestEndpoints {
```

Comment:
maybe better to call ITestApi, ITestResultApi - with 'Api' ending instead of plural 'Endpoints'
for me, 'endpoints' correlate with the list of strings, but it is subjective

Reply:
Not sure, API is something more common, like Aquality Tracking API. Endpoints say that it is a part of something bigger.